ci/jenkins/README.md

```diff
@@ -80,3 +80,7 @@ cloud will be automatically updated with the new changes.
 - `nephe-test-e2e-azure-for-pull-request`: Run end-to-end Azure tests on a Kind cluster with Azure VMs
 - `nephe-test-e2e-eks-for-pull-request`: Run end-to-end AWS tests on EKS cluster with AWS VMs
 - `nephe-test-e2e-aks-for-pull-request`: Run end-to-end Azure tests on AKS cluster with Azure VMs
+- `nephe-test-e2e-aks-linux-agent-for-pull-request`: Run end-to-end AWS tests on EKS cluster with AWS VMs with Linux agent installed.
```

Comment:
/nephe-test-e2e-agentless. It should test aws, azure, eks, and aks
/nephe-test-e2e-agentless-upgrade. It should test aws, azure
/nephe-test-e2e-agented. EKS and AKS for Linux and Windows
/nephe-test-e2e-aws-upgrade, /nephe-test-e2e-azure-upgrade For upgrade
/nephe-test-e2e-all. test everything